### PR TITLE
feat(site-headers): EU, adapting templates, adding demos - front-1285

### PR DESCRIPTION
**Pull request by planctus**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] I have put the vanilla component as `devDependencies`
- [ ] I have put the specs package as `devDependencies`
- [ ] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
- [ ] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
- [ ] My variables naming follow the guidelines (snake case for twig)
- [ ] I have provided tests
- [ ] I have provided documentation (for the "notes" tab)
- [ ] If my local `yarn.lock` contains changes, I have committed it
- [ ] I have given my PR the proper label (`pr: review needed` to indicate that I'm done and now waiting for a review ,`pr: wip` to indicate that I'm actively working on it ...)

### CODE REVIEW

File: src/ec/packages/ec-component-site-header-standardised/__snapshots__/site-header-standardised.test.js.snap [hunk 1]
```diff
@@ -13,17 +14,9 @@ exports[`EC - Site Header Standardised Default renders correctly 1`] = `
...
-        >
-          <img
-            alt="European Commission logo"
-            class="ecl-site-header-standardised__logo-image"
-            src="/logo--en.svg"
-            title="European Commission"
-          />
-        </a>
+          href=""
+        />
```
Comment (papegaill) [inline src/ec/packages/ec-component-site-header-standardised/__snapshots__/site-header-standardised.test.js.snap:19]: no more link, aria-label here seems strange.

File: src/ec/packages/ec-component-site-header/__snapshots__/site-header.test.js.snap [hunk 2]
```diff
@@ -21,7 +21,7 @@ exports[`EC - Site Header Default renders correctly 1`] = `
           <img
             alt="European Commission logo"
             class="ecl-site-header__logo-image"
-            src="/logo--en.svg"
+            src=""
```
Comment (papegaill) [inline src/ec/packages/ec-component-site-header/__snapshots__/site-header.test.js.snap:24]: same here no logo path seems strange.

Comment (planctus) [reply src/ec/packages/ec-component-site-header/__snapshots__/site-header.test.js.snap:24]: Fixed it ;)

File: src/ec/packages/ec-component-site-header-core/README.md [hunk 3]
```diff
@@ -14,7 +14,8 @@ npm install --save @ecl-twig/ec-component-site-header-core
   - "title" (string) (default: ''): Logo title attribute.
   - "alt" (string) (default: ''): Logo alt attribute.
   - "href" (string) (default: ''): Logo URL.
-  - "src" (string) (default: ''): Logo image file path, eg. dist/images/logo--en.svg.
+  - "src_desktop": (string) (default: ''): Desktop logo image file path
+  - "src_mobile": (string) (default: ''): Mobile logo image file path
```
[the remark this answers is not in the document]
Comment (planctus) [reply src/ec/packages/ec-component-site-header-core/README.md:18]: Actually the deprecated one doesn't have any mobile version. I started working on it presuming it was the same as the others and then i noticed it was not, still i left something "misaligned", i fixed that. 
In general, by the way, we only added a parameter (src_mobile) and renamed src in src_desktop (there is no usage of these templates so it's safe), in EC we don't pass the src_mobile, so there shouldn't be any "wrong" logo in EC in mobile.
The logic is that for EC we need the generic ecl-site-header-core__logo-image class, while in EU we need ecl-site-header-core__logo-image--desktop and ecl-site-header-core__logo-image--mobile, both are set to make everything work in the two systems.

[the remark this answers is not in the document]
Comment (planctus) [reply src/ec/packages/ec-component-site-header-core/README.md]: We only added a parameter (src_mobile) and renamed src in src_desktop (there is no usage of these templates so it's safe), in EC we don't pass the src_mobile, so there shouldn't be any "wrong" logo in EC:
```
{% include '@ecl-twig/ec-component-site-header-harmonised/ecl-site-header-harmonised.html.twig' with 
{
logo: {
title: "European Commission",
alt: "European Commission logo",
href: "/example",
language: "en",
src_desktop: "static/media/logo--en.30b933cc.svg"
},
```
I pushed an "update" on the docs, we mention now that the parameter is for EU only. 

For the rest the logic is that for EC we need the generic `ecl-site-header-core__logo-image` class, while in EU we need `ecl-site-header-core__logo-image--desktop` and `ecl-site-header-core__logo-image--mobile`, both are set to make everything work in the two systems.

File: src/ec/packages/ec-component-site-header-standardised/__snapshots__/site-header-standardised.test.js.snap (hunk 1 above)
Comment (planctus) [reply src/ec/packages/ec-component-site-header-standardised/__snapshots__/site-header-standardised.test.js.snap]: yeah, it was wrong the template, only of the standardised, fixed it ;)

**Pull request merged**